### ISSUE TITLE
Improve `Layout/SpaceAroundOperators` cop

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12337]
## Description

https://rubystyle.guide/#spaces-operators

## Describe the solution you'd like

```ruby
# bad
o_scale = 1 / 48r

# good
o_scale = 1/48r
```
